common/src/main/java/com/wynntils/models/character/PlayerInventoryModel.java

```diff
@@ -5,23 +5,37 @@
 package com.wynntils.models.character;
 
 import com.wynntils.core.components.Model;
+import com.wynntils.core.components.Models;
 import com.wynntils.mc.event.ContainerSetContentEvent;
 import com.wynntils.mc.event.SetSlotEvent;
+import com.wynntils.models.items.items.gui.IngredientPouchItem;
 import com.wynntils.models.worlds.event.WorldStateEvent;
 import com.wynntils.models.worlds.type.WorldState;
 import com.wynntils.utils.mc.McUtils;
+import com.wynntils.utils.type.CappedValue;
 import com.wynntils.utils.wynn.InventoryUtils;
 import java.util.List;
 import java.util.Objects;
+import java.util.Optional;
+import net.minecraft.world.item.ItemStack;
 import net.minecraftforge.eventbus.api.SubscribeEvent;
 
 public final class PlayerInventoryModel extends Model {
-    private int openSlots = 0;
+    private static final int MAX_INVENTORY_SLOTS = 28;
```

Comment:
Do we count offhand as a slot or how does this work?

Reply:
No, this is pure inventory slots. Same code basically as before, only the `28` was hard-coded in the calculations. When the current slots == max slots, then you cannot pick up any more items.